test/unit/test-purifier.js

```diff
@@ -432,7 +435,7 @@ function runSanitizerTests() {
       expect(purify('<amp-img [x]="y"></amp-img>', true)).to.match(
           /<amp-img i-amphtml-key="(\d+)" data-amp-bind-x="y" i-amphtml-binding=""><\/amp-img>/);
       // Other elements should NOT have i-amphtml-key-set.
-      expect(purify('<p></p>')).to.equal('<p></p>');
+      expect(purify('<p></p>', true)).to.equal('<p></p>');
```

Comment:
Why add `true` here?

Reply:
the test states that diffing should be set to `true` in this test. Which is not the case when it is not provided, as it defaults to false, though it does not affect the outcome of the test given that the element is not an amp-component.